Eggstravaganza/Assets/Scripts/ItemsSpawn.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using UnityEditor;
+using UnityEngine;
+using Random = UnityEngine.Random;
+public class ItemsSpawn: MonoBehaviour
+{
+    GameObject[] m_Prefabs;
+    [SerializeField]
+    int numItemsToSpawn = 5;
```

Comment:
Can this be something like `maxNumItemsToSpawn`, then it randomly selects the number of objects to spawn between  and the max? (If you have time, just might be a bit more funny)